app-springboot/src/main/java/org/keycloak/quickstart/springboot/service/ProductService.java

```diff
@@ -27,16 +27,8 @@
 @Component
 public class ProductService {
 
```

Comment:
Why are these changes required? Is this indicating some breaking changes in our adapters?